### PR TITLE
Add Evaluation License v2

### PR DESCRIPTION
**Pull request by dtuite**
https://app.shortcut.com/larder/story/5252/update-evaluation-licence-to-include-ability-to-retain-data-for-other-purposes

Preview: https://deploy-preview-511--roadie.netlify.app/legal-notices/evaluation-license/v2/

### CODE REVIEW

File: content/legal-notices/evaluation-license/v2.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: Evaluation Licence
+lastUpdated: '2021-06-06T21:00:00.0Z'
```
Comment (Irma12) [inline content/legal-notices/evaluation-license/v2.md:3]: do we need to put new date here?

File: content/legal-notices/evaluation-license/v2.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+6.2	If Customer wishes to act as data controller for the Trial Period, with Roadie as data processor, please contact Roadie to request Roadie’s Data Processing Agreement.
+
+6.3 Roadie will retain Customer’s data, including personal data, for the Retention Period. Usage data collected during the trial may also be retained after the trial for the Retention Period. Customer data will be retained for the purpose of enabling the Customer to re-start the evaluation or convert to a full license without re-entering the data from the evaluation. Usage data will be retained for the purposes of improving and optimizing the Service and Website. At the end of this Retention Period, where the Customer has not opted to purchase a full licence, Roadie will delete all Customer data on the Service. After the Retention Period Usage data will be retained in pseudo anonymised form for this purpose. This clause overrides the terms of the Data Processing Agreement, if applicable.
```
Comment (oblongcoffee) [inline content/legal-notices/evaluation-license/v2.md:63]: gap missing between 6.3 and start of paragraph

Comment (oblongcoffee) [reply content/legal-notices/evaluation-license/v2.md:63]: spacing is missing too. it is supposed to be three sections so its easier to read. This is pretty dense and hard to read.

Comment (dtuite) [reply content/legal-notices/evaluation-license/v2.md:63]: I chose not to add a gap because no other term has a gap in it. Is the gap important?

Comment (dtuite) [reply content/legal-notices/evaluation-license/v2.md:63]: I'm concerned that it looks like a new clause with no number if we leave gaps.

Comment (oblongcoffee) [reply content/legal-notices/evaluation-license/v2.md:63]: we don't have to add spacing in the paragraph, it just makes it easier to read if we do, but its not essential. 

But the other gap was this:
Screenshot 2021-12-09 at 11 11 11

Comment (dtuite) [reply content/legal-notices/evaluation-license/v2.md:63]: That gap is gone and/or doesn't render in the finished product. [Check the preview to see.](https://deploy-preview-511--roadie.netlify.app/legal-notices/evaluation-license/v2/)

**Review by oblongcoffee — APPROVED** [review]

**Pull request merged**